README.md

```diff
@@ -1,12 +1,16 @@
-# Saliency Methods
+# Updates
```

Comment:
Updates as the main heading doesn't look right maybe rename this to "Saliency Library" and just list updates on top or have a subtitle (## Updates)?

Reply:
Done.